src/themes/default/globals/variation.variables

```diff
@@ -549,6 +549,21 @@
 /* Embed */
 @variationEmbedRatio: true;
 
+/* Flyout */
+@variationFlyoutColumnWidth: true;
+@variationFlyoutThin: true;
+@variationFlyoutWide: true;
+@variationFlyoutTop: true;
+@variationFlyoutBottom: true;
+@variationFlyoutLeft: true;
+@variationFlyoutRight: true;
+@variationFlyoutOverlay: true;
+@variationFlyoutSlideAlong: true;
+@variationFlyoutSlideOut: true;
+@variationFlyoutScale: true;
+@variationFlyoutPush: true;
+@variationFlyoutUncover: true;
+
```

Comment:
You removed everything but `overlay` in the JS code by 4ebcf020dbf2da8839e9e61a7bbc7399deae5100, so there is no support for `push`, `uncover`, `slide along`, `slide out` or `scale` anymore.
Please remove all of the related variables for those unsupported transitions as well.

Reply:
Resolved by deb1a561dfc99c26494d0e5c503c1118c7d4ccd3.